lib/RecommendationProvider.js

```diff
@@ -13,13 +13,14 @@ const DEFAULT_TIMEOUTS = {
 Cu.import("resource://gre/modules/Task.jsm");
 Cu.importGlobalProperties(["fetch"]);
 
-function RecommendationProvider(previewProvider, options = {}) {
+function RecommendationProvider(previewProvider, tabTracker, options = {}) {
```

Comment:
I was going to land that bit (found here https://github.com/mozilla/activity-stream/pull/906/files#diff-cd9b1d133214f36f1c20099d7f6817e6R115) in a separate PR because no one was instantiating RecommendationProvider yet, but that patch landed so I can add it to this PR